tests/unit/tree/test_repo.py

```diff
@@ -422,11 +422,15 @@ def test_subrepo_walk(tmp_dir, scm, dvc, dvcfiles, extra_expected):
     ]
 
     actual = []
-    for root, dirs, files in tree.walk("dir", dvcfiles=dvcfiles):
+    for root, dirs, files in tree.walk(
```

Comment:
This used to use LocalTree for subrepos, which was wrong. Using GitTree now, which always yields abs paths.